### PR TITLE
NTR-substantia-nigra-dopaminergic-neuron-for-Basal-Ganglia-BICAN

### PR DESCRIPTION
**Pull request by AvolaAmg**
terms added: substantia nigra dopaminergic neuron

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32370,6 +32372,15 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (substantia nigra dopaminergic neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32826893") Annotation(oboInOwl:hasDbXref "PMID:35308118") obo:IAO_0000115 obo:CL_4042023 "A midbrain dopaminergic neuron that has its soma located in a substantia nigra. This dopaminergic neuron type is metabolically active and it is involved in the regulation of movement, cognition, motivation and reward. Neurodegeneration of this dopaminergic neuronal type causes loss in fine motor control in Parkinson's Disease.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32377]: This dopaminergic neuron type is metabolically active
-->
This dopaminergic neuron type is highly metabolically active

**Review by dosumis — COMMENTED** [review]
I guess not quite ready for review as lacks classification and location.  

NOTE - clashing ID with https://github.com/obophenotype/cell-ontology/pull/2569/files

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32378,6 +32380,15 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (substantia nigra dopaminergic neuron)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32383]: ID CLASH

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -32378,6 +32380,15 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32826893") Annotation(oboInOwl:hasDbXref "PMID:35308118") obo:IAO_0000115 obo:CL_4042023 "A midbrain dopaminergic neuron that has its soma located in a substantia nigra. This dopaminergic neuron type is highly metabolically active and it is involved in the regulation of movement, cognition, motivation and reward. Neurodegeneration of this dopaminergic neuronal type causes loss in fine motor control in Parkinson's Disease.")
+AnnotationAssertion(terms:contributor obo:CL_4042023 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042023 "2024-09-16T13:09:29Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:SynonymTypeProperty obo:OMO_0003000) Annotation(oboInOwl:hasDbXref "PMID:32826893") oboInOwl:hasBroadSynonym obo:CL_4042023 "DaN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35308118") oboInOwl:hasNarrowSynonym obo:CL_4042023 "SNpc dopaminergic neuron")
+AnnotationAssertion(rdfs:label obo:CL_4042023 "substantia nigra dopaminergic neuron"@en)
+
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32391]: Classification, location etc still needed

**Review by dosumis — COMMENTED** [review]
Good so far.  See comments for what's still needed

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32383]: will be resolved as the branch is merged.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32391]: Could you be more specific? 
the definition includes:
> A midbrain dopaminergic neuron that **has its soma located in a substantia nigra** . This dopaminergic neuron type is **highly metabolically active** and **it is involved in** the regulation of movement, cognition, motivation and reward. Neurodegeneration of this dopaminergic neuronal type causes loss in fine motor control in Parkinson's Disease.

for classification see the most recent commit.

**Comment by dosumis** [comment]
#gogoeditdiff

**Review by dosumis — APPROVED** [review]
Better to assert location and autoclassify - but in this case the general classification used for the grouping class might be a problem for that.  Minor issue so approving as the term is needed.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -32388,6 +32390,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12429182") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4042021 "neuronal-restricted precursor"@en)
 SubClassOf(obo:CL_4042021 obo:CL_0000123)
 SubClassOf(obo:CL_4042021 obo:CL_0011026)
+SubClassOf(obo:CL_4042021 obo:CL_2000097)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32393]: Is this a stray edit on a different term?  Looks a bit odd.

**Review by dosumis — CHANGES_REQUESTED** [review]
Main term looks good + good to see file reserialised.  Please check the odd stray subclassOf axiom in the diff.  Should it be there?

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32393]: all done

**Review by dosumis — APPROVED** [review]

**Pull request merged**